README

```diff
@@ -0,0 +1,7 @@
+Inside the directory run:
+
+docker-compose up -d
+
+Run the following to perform migrations:
+
+docker run -it --rm --link flywaypostgres_postgres_1 -v `pwd`:/flyway/sql addteq/flyway -url=jdbc:postgresql://flywaypostgres_postgres_1:5432/flyway -user=postgres -password=mysecretpassword migrate
```

Comment:
@donaldchampion docker-compose run sounds better, else it defeats the purpose of using compose https://docs.docker.com/compose/reference/run/ You don't need to worry about host mounted volume.